powershell-adapter/psDscAdapter/psDscAdapter.psm1

```diff
@@ -67,7 +67,7 @@ function FindAndParseResourceDefinitions
         return
     }
     
-    "Loading resources from '$filePath'" | Write-DscTrace -Operation Trace
+    "Loading resources from file '$filePath'" | Write-DscTrace -Operation Trace
```

Comment:
Not a blocker, but using the pipeline is inherently slower than using the parameter to the helper function directly as it forces use of the parameter binder

Reply:
Pipeline looks cooler though :)